styles/styles.css

```diff
@@ -203,7 +203,7 @@
 }
 
 html {
-  font-size: 10px;
```

Comment:
Please, don't change this in Mack, if we do so, it should be done wrapped by `redesign-v2` ;)